packages/compiler-cli/src/ngtsc/translator/src/translator.ts

```diff
@@ -387,6 +388,11 @@ export class ExpressionTranslatorVisitor<TStatement, TExpression> implements o.E
       T {
     return this.factory.setSourceMapRange(ast, createRange(span));
   }
+
+  private attachComments(statement: TStatement, leadingComments: LeadingComment[]): TStatement {
```

Comment:
Can this be

```suggestion
  private attachComments(statement: TStatement, leadingComments: o.LeadingComment[]): TStatement {
```

and remove the (deep) import?